config/setup/browse.go

```diff
@@ -193,6 +193,10 @@ th a {
 		margin-top: 70px;
 	}
 }
+
+#name {
+	white-space: pre;
```

Comment:
Does this cause wrapping problems with long file names? Maybe we should use pre-wrap instead?